### ISSUE TITLE
Icelandic words with no features

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/UD_Icelandic-IcePaHC#5]
There are some cases where a word which generally has features is labeled without features.  For example, the pronoun `mín` is usually featurized similar to this when in the genitive case:

```
38      mín     ég      PRON    PRO-G   Case=Gen|Number=Sing|Person=1|PronType=Prs      33      obl     _       IFD_tag=fp1ee|SpaceAfter=No
```

whereas it has blank features in the following example:

```
# sent_id = 1275.MORKIN.NAR-HIS,729.4956
# X_ID = 1275.MORKIN.NAR-HIS,.1445
# text = " Nei," segir konungur," yrk nú til mín."
12      mín     ég      PRON    PRO-G   _       9       obl     _       IFD_tag=x|SpaceAfter=No
```

I came across this when I was looking for some instances of lemmas where the correct lemma form of a word is ambiguous - the same word & POS tag can have multiple resolutions.  An example in English is `'s`, which can be either `be` or `have` depending on the context.

In this treebank, there are a few examples, notably:

```
mín PRON Counter({'minn': 322, 'ég': 154})
sín PRON Counter({'sinn': 260, 'sig': 200})
```

It would seem though that there is a clear distinction between the xpos of PRO-A or PRO-G.  Is that a correct assessment?

There are a couple more imbalanced examples:

```
þeim PRON Counter({'það': 1765, 'hann': 96, 'sá': 12, 'hún': 2})
þeirra PRON Counter({'hann': 1306, 'það': 82, 'hún': 32, 'sá': 5, 'þeir': 1})
```

There are a variety of possible features here, and the xpos tags are generally the same, so it looks like this isn't easy to distinguish like the `mín` case was

```
14      þeim    það     PRON    PRO-D   Case=Dat|Gender=Masc|Number=Sing|PronType=Dem   15      nmod    _       IFD_tag=fakeþ
13      þeim    það     PRON    PRO-D   Case=Dat|Gender=Masc|Number=Plur|PronType=Prs   11      appos   _       IFD_tag=fpkfþ
17      þeim    hann    PRON    PRO-D   Case=Dat|Gender=Masc|Number=Plur|PronType=Prs   15      obj     _       IFD_tag=fpkfþ|SpaceAfter=No
134     þeim    það     PRON    PRO-D   Case=Dat|Gender=Masc|Number=Plur|PronType=Prs   132     appos   _       IFD_tag=fpkfþ
15      þeim    hann    PRON    PRO-D   Case=Dat|Gender=Masc|Number=Sing|PronType=Dem   10      iobj    _       IFD_tag=fakeþ
28      þeim    hann    PRON    PRO-D   Case=Dat|Gender=Masc|Number=Plur|PronType=Prs   26      obl     _       IFD_tag=fpkfþ|SpaceAfter=No
```

Would it be possible to request an explanation of the difference here?  I was thinking that it would be an interesting project to write up a classifier which can distinguish the two lemma possibilities.  Thanks in advance!

**Comment by einarfs** [comment]
Thank you!

Regarding the issue "There are some cases where a word which generally has features is labeled without features": I will work on this for the next release - I will try to identify all these. I had fixed some but it will need more systematic checking on my part.

"It would seem though that there is a clear distinction between the xpos of PRO-A or PRO-G. Is that a correct assessment?" Yes, PRO-A is a pronoun in the accusative and PRO-G a pronoun in the genitive. In the example you show, "til mín" 'to me', "til" is a preposition taking the genitive form of 1.sg. personal pronoun. In something like "Ég sá börnin mín" 'I saw my children', "mín" is an accusative form of the 1.sg. possessive pronoun.

For "þeim" and "þeirra": there is a systematic syncretism in the dative plural on the one hand and genitive plural on the other in, e.g., adjectives and third person pronouns. In addition, "þeim" and "þeirra" can both be a personal pronoun or a demonstrative.

I would be happy to work more on this.

**Comment by AngledLuffa** [comment]
Great, thanks for the quick response!

In terms of `þeim` and `þeirra`, I see two different lemmas with no distinction in the upos or xpos.  Is that correct?  If so, we'll try to build a classifier that can distinguish the two cases.

**Comment by einarfs** [comment]
Yes, that's right, and there are also cases where PRON þeim or þeirra will be lemmatized as hún 'she'.
